stations.csv

```diff
@@ -19445,41 +19445,41 @@ id;name;slug;uic;uic8_sncf;longitude;latitude;parent_station_id;is_city;country;
 21011;Bosco Redole;bosco-redole;8309473;;14.5879;41.4593;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8309473;t;;f;;;;;;;;;;;;;;;;;;
 21012;Arcore;arcore;8301510;;9.32292;45.6238;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8301510;t;;f;;;;;;;;;;アルコレ;;;;;;;;
 21013;Ardara;ardara;8312817;;8.81849;40.6321;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8312817;t;;f;;;;;;;;;;;;;;;;;;
 21014;Ardenno-Masino;ardenno-masino;8301427;;9.65042;46.1617;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8301427;t;;f;;;;;;;;;;;;;;;;;;
 21015;Bortigiadas;bortigiadas;8343923;;9.05639;40.8919;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8343923;t;;f;;;;;;;;;;;;;;;;;;
-21016;Bosa Marina;bosa-marina;8342502;;;;;;IT;f;Europe/Rome;f;;f;;f;;f;;f;;f;8342502;t;;f;;;;;;;;;;;;;;;;;;
-21017;Anfiteatro;anfiteatro;8352006;;;;;;IT;f;Europe/Rome;f;;f;;f;;f;;f;;f;8352006;t;;f;;;;;;;;;;;;;;;;;;
-21018;Colosimi;colosimi;8354214;;;;;;IT;f;Europe/Rome;f;;f;;f;;f;;f;;f;8354214;t;;f;;;;;;;;;;;;;;;;;;
+21016;Bosa Marina;bosa-marina;8342502;;8.4794525;40.2900509;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8342502;t;;f;;;;;;;;;;;;;;;;;;
+21017;Anfiteatro;anfiteatro;8352006;;14.2448574;41.0860659;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8352006;t;;f;;;;;;;;;;;;;;;;;;
+21018;Colosimi;colosimi;8354214;;16.3954772;39.1190633;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8354214;t;;f;;;;;;;;;;;;;;;;;;
 21019;Comignago;comignago;8300050;;8.56611;45.7143;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8300050;t;;f;;;;;;;;;;;;;;;;;;
 21020;Comitini;comitini;8312212;;13.653;37.4238;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8312212;t;;f;;;;;;;;;;;;;;;;;;
 21021;Cappelle-Magliano;cappelle-magliano;8308525;;13.3724;42.0596;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8308525;t;;f;;;;;;;;;;;;;;;;;;
 21022;Consandolo;consandolo;8327230;;11.7758;44.8297;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8327230;t;;f;;;;;;;;;;;;;;;;;;
-21023;Como Nord Camerlata;como-nord-camerlata;8325432;;;;;;IT;f;Europe/Rome;f;;f;;f;;f;;f;;f;8325432;t;;f;;;;;;;;;;;;;;;;;;
-21024;Como Nord Lago;como-nord-lago;8325435;;;;;;IT;f;Europe/Rome;f;;f;;f;;f;;f;;f;8325435;t;;f;;;;;;;;;;;;;;;;;;
-21025;Bisuschio-Viggiu’;bisuschio-viggiu;8301202;;;;;;IT;f;Europe/Rome;f;;f;;f;;f;;f;;f;8301202;t;;f;;;;;;;;;;;;;;;;;;
+21023;Como Nord Camerlata;como-nord-camerlata;8325432;;9.079455;45.7844993;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8325432;t;;f;;;;;;;;;;;;;;;;;;
+21024;Como Nord Lago;como-nord-lago;8325435;;9.0840647;45.8141057;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8325435;t;;f;;;;;;;;;;;;;;;;;;
+21025;Bisuschio-Viggiu’;bisuschio-viggiu;8301202;;8.8875605;45.8707099;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8301202;t;;f;;;;;;;;;;;;;;;;;;
 21026;Bitetto-Palo del Colle;bitetto-palo-del-colle;8311501;;16.7438;41.0457;;;IT;f;Europe/Rome;t;;f;;f;;f;;f;;f;8311501;t;;f;;;;;;;;;;;;;;;;;;
-21027;Boccia al Mauro;boccia-al-mauro;8309760;;;;;;IT;f;Europe/Rome;f;;f;;f;;f;;f;;f;8309760;t;;f;;;;;;;;;;;;;;;;;;
+21027;Boccia al Mauro;boccia-al-mauro;8309760;;14.500691;40.789348;;;IT;f;Europe/Rome;f;;f;;f;;f;;f;;f;8309760;t;;f;;;;;;;;;;;;;;;;;;
```

Comment:
If the station is actually closed, disable it for trenitalia. Maybe one day there will be a private operator re-opening the line